lib/OpenQA/Worker/Job.pm

```diff
@@ -1080,12 +1079,8 @@ sub _upload_log_file {
         my $tx = $ua->build_tx(POST => $ua_url => form => $upload_parameter);
 
         if ($regular_upload_failed) {
-            log_warning(
-                sprintf(
-                    'Upload attempts remaining: %s/%s for %s, in %s seconds',
-                    $retry_counter--, $retry_limit, $filename, $tics
-                ));
-            OpenQA::Utils::wait_with_progress($tics);
+            log_warning(sprintf('Upload attempts remaining: %s/%s for %s', $retry_counter--, $retry_limit, $filename));
+            sleep 1;
```

Comment:
This was waiting 5 seconds before. I wouldn't change that. 

It would make sense to make the magic numbers configurable via env variables. There's already `OPENQA_UPLOAD_DELAY` which is read into a constant within the same file so one could just use that constant.

Reply:
Resolved.